packages/pkg-config-openbsd/pkg-config-openbsd.1.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "http://www.freedesktop.org/wiki/Software/pkg-config/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL"
+build: [
+  ["pkgconf" "--help"]
+]
+install: [
+  ["ln" "-s" "/usr/local/bin/pkgconf" "%{bin}%/pkg-config"]
+]
+remove: [
+  ["rm" "-f" "%{bin}%/pkg-config"]
+]
+post-messages: [
+  "pkg-config-openbsd: A symlink to /usr/local/bin/pkgconf has been installed in the OPAM bin directory (%{bin}%) on your PATH as 'pkg-config'." {os = "openbsd"}
+]
+depexts: [
+  ["devel/pkgconf"]
+]
+available: [os = "openbsd"]
+synopsis: "OpenBSD fix for conf-pkg-config"
+description: """
+This package can only install if the pkgconf package is installed
```

Comment:
Erroneous (copied) description ?

Reply:
Can't you edit the PR if you spot a typo?

Reply:
What is the correction that you are suggesting?

Reply:
It was just about the description that was not the right one for this package. Updated